README.md

```diff
@@ -225,6 +225,12 @@ $ nix-env --install ripgrep
 $ # (Or using the attribute name, which is also `ripgrep`.)
 ```
 
+If you're an **Ubuntu** user and you are using `16.04 LTS` or later, you can install `ripgrep` from the `snap` store:
```

Comment:
Could you please wrap this to 79 columns inclusive like the rest of the README?

Also, do Ubuntu users need to install `snap` themselves, or is it on 16.04 by default?

Finally, could you remove the reference to Ubuntu above? (Since we know show how to install it.)

Reply:
Yes of course. I wrapped the lines to 79 and add some clarifications for the Ubuntu versions. `snap` is installed by default in Ubuntu 16.04 and later versions.

What I didn't understand is what do you mean with this:
> Finally, could you remove the reference to Ubuntu above? (Since we know show how to install it.)

Thanks for the feedback! :)

Reply:
Just search for mentions of `Ubuntu`. The README says that Ubuntu isn't mentioned below, but this PR adds instructions for Ubuntu, so that section is now outdated. :-)